cmd/earthly/satellite_cmds.go

```diff
@@ -302,7 +304,15 @@ func (app *earthlyApp) actionSatelliteSelect(cliCtx *cli.Context) error {
 	app.commandName = "satelliteSelect"
 
 	if cliCtx.NArg() != 1 {
-		return errors.New("satellite name is required")
+		if app.cfg.Satellite.Name == "" {
+			app.console.Printf("No satellite selected\n\n")
+			app.console.Printf("To select a satellite:\n\t%s\n", selectUsageText)
+		} else {
+			app.console.Printf("Selected satellite: %s\n\n", app.cfg.Satellite.Name)
+			app.console.Printf("To select a different satellite:\n\t%s\n", selectUsageText)
```

Comment:
Is this handling the case where a user is re-selecting the satellite with the same name?

Reply:
I ask because it sounds like that what is intended in the PR description:

> Select with a satellite already selected

but the code looks to me like its handling the case where multiple satellites are selected at once. E.g.: 

```
earthly satellite select my-sat-1 my-sat-2
```

Apologies if I'm misunderstood something

Reply:
The behaviour of re-selecting a satellite with the same name will remain the same; it's idempotent and the selection will remain the same.

> Select with a satellite already selected

oooooohh sorry if this was confusing. The text output was slightly different depending on if one was selected or not.
If none are selected, it will output `No satellite selected` in the screenshot. If there is one selected it will print out its name `Selected satellite: local-dev` in the screenshot.

Reply:
ah, I get what you mean now 😄  - I totally mis-read that, not sure why I had it in my head that it was for a satellite _with the same name_